src/components/Settings/index.jsx

```diff
@@ -66,29 +69,40 @@ export class Settings extends Component {
       instanceQuery,
       contextQuery,
       storageData,
-      settingsAppURL
+      settingsAppURL,
+      isFetching
     } = this.props
-    const isCurrentlyFetching = isFetching([
-      diskUsageQuery,
-      instanceQuery,
-      contextQuery
-    ])
+
     let shoulDisplayViewOfferButton = false
     let managerUrlPremiumLink
-    if (!isCurrentlyFetching) {
-      const data = {
-        context: contextQuery,
-        diskUsage: diskUsageQuery,
-        instance: instanceQuery
+    let isFetchingFromQueries
+    const canCheckPremium = cozyClientCanCheckPremium()
+    if (canCheckPremium) {
+      isFetchingFromQueries = isFetchingQueries([
+        diskUsageQuery,
+        instanceQuery,
+        contextQuery
+      ])
+      if (!isFetchingFromQueries) {
+        const data = {
+          context: contextQuery,
+          diskUsage: diskUsageQuery,
+          instance: instanceQuery
+        }
+        shoulDisplayViewOfferButton = instanceModel.shouldDisplayOffers(data)
```

Comment:
s/shoulDisplayViewOfferButton/shouldDisplayViewOfferButton

Reply:
same again and again 🤦‍♂